packages/boxel/addon/components/boxel/action-chin/action-status-area/index.gts

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
```

Comment:
It's not really correct that glint is pushing us to convert from a template-only component to a glimmer component. Those are not the same thing.

We should be able to drop the wrapping class and just have a bare `<template></template>` here, and that is legal and runs but glint doesn't provide us a place to put component signature in that case. I filed https://github.com/typed-ember/glint/issues/387 to ask about this.

Reply:
I got help from the dev-typescript discord channel and resolved this.